### DISCUSSION TITLE
New m5stickc plus 2

### DISCUSSION THREAD

**Discussion by luca320** [raomin/ESPAltherma#397] [General]
Ciao a tutti. Premetto che utilizzo da anni una m5stickc, che ho programmato con il software visual studio. Non ho avuto alcun problema. Letture precise di tutti i sensori espaltherma. Ho comprato da poco una nuova m5stickc plus 2, che ho pensato di tenere di scorta nel caso la prima chiavetta dovesse non funzionare più.  Ho formattato la m5stickcplus usando m5burner e uniflow dedicato al modello corretto, ho fatto upload del progetto selezionando la porta com corretta e come modello, non essendoci la plus 2, ho messo m5stickc plus. la procedura ha esito positivo, non ci sono errori e la scrittura sulla chiavetta va a buon fine. però non riesco a vederla online all'interno dei dispositivi collegati al router. Quale può essere il problema? Ringrazio chi si prenderà il tempo di rispondermi.

Hi everyone. Let me start by saying that I have been using an m5stickc for years, which I programmed with the visual studio software. I didn't have any problems. Precise readings of all espaltherma sensors. I recently bought a new m5stickc plus 2, which I thought I'd keep as a spare in case the first stick no longer works. I formatted the m5stickcplus using m5burner and uniflow dedicated to the correct model, I uploaded the project by selecting the correct com port and as the model, since there was no plus 2, I put m5stickc plus. the procedure is successful, there are no errors and the writing on the flash drive is successful. but I can't see it online within the devices connected to the router. What could be the problem? I thank those who take the time to answer me.

**Comment by MarkLofthouse** [comment]
Same here - found another thread that says the code doesn’t work on the Plus 2, can we get a view if it’s in the roadmap before I return the plus 2? Thanks.
